Applications/Utils/MeshEdit/ResetPropertiesInPolygonalRegion.cpp

```diff
@@ -0,0 +1,216 @@
+/*
+ * \date 2014-09-25
+ * \brief Reset material properties in meshes in a polygonal region.
+ */
+
+// TCLAP
+#include "tclap/CmdLine.h"
+
+// ThirdParty/logog
+#include "logog/include/logog.hpp"
+
+// BaseLib
+#include "BaseLib/LogogSimpleFormatter.h"
+
+// FileIO
+#include "FileIO/readMeshFromFile.h"
+#include "FileIO/XmlIO/Boost/BoostXmlGmlInterface.h"
+#include "FileIO/VtkIO/VtuInterface.h"
+
+// GeoLib
+#include "GeoLib/GEOObjects.h"
+#include "GeoLib/Polygon.h"
+#include "GeoLib/AnalyticalGeometry.h"
+
+// MathLib
+#include "MathLib/Vector3.h"
+
+// MeshLib
+#include "MeshLib/Mesh.h"
+#include "MeshLib/Node.h"
+#include "MeshLib/Elements/Element.h"
+#include "MeshLib/MeshEditing/ElementValueModification.h"
+
+GeoLib::Polygon rotatePolygonToXY(GeoLib::Polygon const& polygon_in,
+	MathLib::Vector3 & plane_normal)
+{
+	// 1 copy all points
+	std::vector<GeoLib::Point*> *polygon_pnts(new std::vector<GeoLib::Point*>);
+	for (size_t k(0); k < polygon_in.getNumberOfPoints(); k++)
```

Comment:
Dima cares about using ``size_t`` with its namespace ``std``. Do we have a rule for this? I think if it is used in cpp file and the file does not include any C header, using size_t directly is OK and less verbose.